api/functions/process_dataset.py

```diff
@@ -37,17 +37,19 @@ def callback(publish_future: pubsub_v1.publisher.futures.Future) -> None:
 
     # Read the required files
     dataset = data['value']
-    print(dataset)  # TODO find errors here.
-    files = dataset['files']
+    print(dataset)
```

Comment:
Is loguru still the right go with this python code running in cloud functions? I know that printing to stdout here is the default way of logging: https://cloud.google.com/functions/docs/monitoring/logging and it seems cloud functions have their own specific way of logging with different severity levels.

Reply:
> > * Strange (and, I think unnecessary) inline import of `base64`
> 
> Hey this is actually something I wanted to ask you about. I see this a lot in the google reference documentation for cloud functions, and I was equally uncomfortable with it. I assumed, and I have no idea how correct this is, that local imports in a particular cloud function are the least impactful on other cloud functions that might run from the same file- do you know anything about this?

I can see two rationales for having the inline import:
- Speed, not importing requirements globally saves time in functions that don't rely on them.
- Some libraries fiddle with the (normally global) closure state on import, which you'd typically want to avoid in a use case like this.

I think we have a long way to go before the first applies to us, and the second doesn't typically apply to a stdlib package, and definitely doesn't in the case of base64 specifically.

Reply:
Fixed in latest revision.